src/instructlab/cli/rag/convert.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+import logging
+import os
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.configuration import DEFAULTS
+from instructlab.rag.process_docs import (
+    process_docs_from_folder,
+    process_docs_from_taxonomy,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "--input_dir",
+    required=False,
+    type=click.Path(exists=True, file_okay=False, dir_okay=True, readable=True),
+    default=None,
+    help="The folder with user documents to process. In case it's missing, the knowledge taxonomy files will be processed instead.",
+)
+@click.option(
```

Comment:
shouldn't we also allow to configure the taxonomy flags with `clickext.ConfigOption`? of course the best would be to reuse the settings under the `generate` section, but we know this is not doable ATM.